Makefile

```diff
@@ -362,7 +362,19 @@ switch-tanzu:
 #################################
 ##  --    Docker images    --  ##
 #################################
+
 docker-multiarch-build: go-generate generate-config-file 
+ifeq ($(SNAPSHOT),true)
```

Comment:
One more little thing, I think it's safer to have the `if` applied to the release build and the `else` to the nightly build, so that if `SNAPSHOT=yolo` or SNAPSHOT is unexpectedly not set, it's considered as a nightly build. So I would invert the logic.

Reply:
Updated.  Let me know if this isn't the inverted logic you had in mind.  I think it's what you're referring to.